app/mailers/contact_mailer.rb

```diff
@@ -7,6 +7,8 @@ def submit_feedback(params, ip)
     @email = params[:email] || 'User not logged in'
     @message = params[:message]
     @ip = ip
+    @barcode = params[:barcode] || nil
+    @status = params[:status] || nil
```

Comment:
```suggestion
    @barcode = params[:barcode]
    @status = params[:status]
```

I'm not sure we need the fallbacks.